assets/prototype/domain/eventEditor/data/queries/datetimes/test/contextWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { MockedProvider, MockedResponse } from '@apollo/react-testing';
```

Comment:
Yes, very cool:
```
The MockedProvider is a test-utility that allows you to create a mocked version of the ApolloProvider that doesn't send out network requests to your API, but rather allows you to specify the exact response payload for a given request.
```

Reply:
And `MockedResponse` is also nice interface: https://github.com/apollographql/react-apollo/blob/master/packages/testing/src/mocks/types.ts